src/erlang_ls_parser.erl

```diff
@@ -266,12 +266,12 @@ attribute(Tree, Extra) ->
         {include, [String]} ->
           [erlang_ls_poi:new( Tree
                             , include
-                            , erl_syntax:string_literal(String)
+                            , erl_syntax:string_value(String)
```

Comment:
`erl_syntax:string_literal/1` was returning `"\"foo\""`, I don't know how we got it working when the value was that 🤷‍♂ .